spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/actions/ExpireSnapshotsSparkAction.java

```diff
@@ -132,23 +129,6 @@ public ExpireSnapshotsSparkAction deleteWith(Consumer<String> newDeleteFunc) {
     return this;
   }
 
-  /**
-   * Expires snapshots and commits the changes to the table, returning a Dataset of files to delete.
-   *
-   * <p>This does not delete data files. To delete data files, run {@link #execute()}.
-   *
-   * <p>This may be called before or after {@link #execute()} to return the expired files.
-   *
-   * @return a Dataset of files that are no longer referenced by the table
-   * @deprecated since 1.0.0, will be removed in 1.1.0; use {@link #expireFiles()} instead.
```

Comment:
Looks good to me.  Looks like comment was wrong, it was actually deprecated as of 1.1 (1.0 did not have deprecation).

But it looks like it still fulfills the deprecation guideline?

Reply:
We probably missed to include it in 1.0, should be safe to drop in 1.2, though!